middle_end/flambda2/from_lambda/closure_conversion.ml

```diff
@@ -623,7 +616,15 @@ let close_c_call acc env ~loc ~let_bound_ids_with_kinds
             Misc.fatal_errorf "Expected one arg for %s" prim_native_name)
         | _, _ ->
           Misc.fatal_errorf "Wrong argument and/or result kind(s) for %s"
-            prim_native_name)
+            prim_native_name
+    in
+    match prim_native_name with
+    | "caml_int64_float_of_bits_unboxed" ->
+      unboxed_int64_to_and_from_unboxed_float ~src_kind:(Unboxed_integer Pint64)
+        ~dst_kind:(Unboxed_float Pfloat64) ~op:Unboxed_int64_as_unboxed_float64
+    | "caml_int64_bits_of_float_unboxed" ->
```

Comment:
It's a bit unfortunate to add another secret builtin. What do you think about making these primitives, or requiring is_c_builtin? 

Ideally, I think we'd assure that %primitives are specialized before the middle end, @@ builtins are optionally specialized after the middle end, and otherwise externals are always C calls.

Reply:
I think I'd prefer to leave the PR as-is, since we already have references to these builtins in (at least) the stdlib, but we could consider changing these to primitives later.

In fact I think some of the `@@builtin`s should probably be recognised by flambda2, but that's a separate discussion.